tests/main/interfaces-framebuffer/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Ensure that the framebuffer interface works.
+
+details: |
+    The framebuffer interface allows to access the /dev/fb* buffer files.
+
+    The test uses the test-snapd-framebuffer snap to write/read from /dev/fb0.
+
+    The test also checks the interface connection and disconnection works properly.
+
+prepare: |
+    snap try $TESTSLIB/snaps/test-snapd-framebuffer
```

Comment:
We should probably snap remove it in restore?

Reply:
Thanks for reviewing this. Snaps are automatically removed in the reset.sh script which is called in the test setup.